tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -34,6 +34,10 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-nocodec.bin,DEEPBUFFER_FW_DMA_MS=100,\
 DEEPBUFFER_D0I3_COMPATIBLE=true"
 
+# SSP topology for LNL FPGA with lower DMIC IO clock of 19.2MHz
+"cavs-nocodec\;sof-lnl-nocodec-fpga\;PLATFORM=lnl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-lnl-nocodec.bin,PASSTHROUGH=true,DMIC_IO_CLK=19200000"
```

Comment:
sof-lnl-nocodec-fpga.bin?

What are the rules for such files?

Reply:
there're not rules. if the names collide, the first one gets overriden.